#Request

geometry_msgs/PoseStamped final_pose
string link_name
bool send

---

#Response

bool response
